src/planner/binder/statement/bind_create.cpp

```diff
@@ -644,8 +643,11 @@ BoundStatement Binder::Bind(CreateStatement &stmt) {
 		}
 		auto entry = config.storage_extensions.begin();
 		auto &storage_extension = entry->second;
+		if (storage_extension->create_database_extension_function == nullptr) {
+			throw BinderException("CREATE DATABASE is not supported in this extension.");
```

Comment:
You have the extension name in `entry->first` so you can use it in the error message

Reply:
Done